examples/visualization/data-observatory/deckgl-boundaries.html

```diff
@@ -0,0 +1,170 @@
+
+  <body>
+    <div id='map'></div>
+    <div id='controls'>
+      <select id='geographies' onchange='changeGeography(this.value)'></select>
+    </div>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8/dist.min.js"></script>
+
+    <script>
+      
+      // // Local environment
+      let host = 'https://aasuero.carto.com/api/v4/data/observatory'
+      
+
+      let initCenter = [-73.989582, 40.741587];  // [-87.623177, 41.881832],
+      let initZoom = 12;
+        
+      let geographies = [
+        {
+          'geography': 'usct_state_4c8090b5',
+          'variable': 'total_pop_624c2d45',
+          'name': 'State',
+          'zoom': 4,
+        },
+        {
+          'geography': 'usct_county_ec40c962',
+          'variable': 'total_pop_6b4a52ef',
+          'name': 'County',
+          'zoom': 5,
+        },
+        {
+          'geography': 'usct_censustract_bc698c5a',
+          'variable': 'total_pop_e7000b7f',
+          'name': 'Tract',
+          'zoom': 7,
+        },
+        {
+          'geography': 'usct_zcta5_75071016',
+          'variable': 'total_pop_ce997f4e',
+          'name': 'Zip Codes',
+          'zoom': 7,
+        },
+        {
+          'geography': 'usct_blockgroup_f45b6b49',
+          'variable': 'total_pop_cc1a00e3',
+          'name': 'Block groups',
+          'zoom': 9,
+        },
+        {
+          'geography': 'usct_block_1e0d4b08',
+          'variable': '',
+          'name': 'Block',
+          'zoom': 12,
+        }
+      ];
+
+      function findGetParameter(parameterName) {
+        var result = null,
+            tmp = [];
+        location.search
+            .substr(1)
+            .split("&")
+            .forEach(function (item) {
+              tmp = item.split("=");
+              if (tmp[0] === parameterName) result = decodeURIComponent(tmp[1]);
+            }); 
+        return result;
+      }
+
+      function drawGeographiesDropDown(){
+        let html = '';
+        for (var opt of geographies){
+          html += `<option value='${opt.geography}'>${opt.name}</option>`;
+        }
+        document.getElementById('geographies').innerHTML = html;
+      }
+      
+      function getURLTemplates(suffix) {
+        const domains = ['a','b','c','d'];
+        const urlTemplate = 'https://do-gusc-{domain}.cartocdn.com/USER/api/v4/data/observatory/';
+        
+        return domains.map(d => urlTemplate.replace('{domain}', d) + suffix);
+
+      }
+
+      async function changeGeography(selectedGeography){
+        const geography = geographies.find((g) => {
+            return g.geography == selectedGeography;
+        });
+        await loadGeography(geography);
+      }
+
+      async function loadGeography(g){
+        data_tiles = {};
+        // map.setZoom(g.zoom);
+        variable = g.variable;
+
+        const mvtLayer = new window.deck.MVTLayer({
```

Comment:
Shouldn't we use our DOSource at all times?

Reply:
No because on this example I just wanted to try the boundaries without the merge with data. This is an internal example that we won't make public, it's just to let data team debug the performance of the API.